ui/apps/platform/src/Containers/NetworkGraph/NetworkGraphPage.tsx

```diff
@@ -123,10 +127,29 @@ function NetworkGraphPage() {
         setSearchFilter(modifiedSearchObject);
     }
 
+    const onBreadcrumbScopeChange = (newFilter: SearchFilter) => {
```

Comment:
I was a little confused by the function name. Would something like `clearGraphOnEmptySelection` be more appropriate?

Reply:
Yeah this is a good suggestion. I tended towards the generic `onBreadcrumbScopeChange`, since it is the event that happens when the scope changes, but that doesn't give a good indication of _what_ happens. If there was more than one thing we wanted to run on change I think the generic choice might be more appropriate.

I'll change this to `clearGraphOnEmptyScope`, which will read nicely when used in the props below:
`onScopeChange={clearGraphOnEmptyScope}`.